lib/kafo/help_builders/base.rb

```diff
@@ -82,7 +82,11 @@ def params_mapping(items)
       end
 
       def parametrization
-        @parametrization ||= Hash[@params.map { |p| [parametrize(p), p] }]
+        @parametrization ||= begin
+          @params.inject({}) do |h,p|
+            h.update(parametrize(p) => p, parametrize(p, 'reset-') => p)
```

Comment:
I hid all of the reset options in the basic help output (default --help) but left them in --full-help for completeness as it might show up for people who grep the output.  I do partly agree though, I was planning to hide them at first and try to add "--reset-[NAME]  Reset parameter NAME" or something instead, but didn't do that in the end.